src/Files.App/ViewModels/UserControls/SidebarViewModel.cs

```diff
@@ -1055,20 +1055,21 @@ private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigatio
 					CommandParameter = menu,
 					ShowItem = options.ShowProperties
 				},
-				new ContextMenuFlyoutItemViewModel()
+				new(ContextFlyoutItemType.Separator)
 				{
-					ItemType = ContextMenuFlyoutItemType.Separator,
+					ShowItem = true,
```

Comment:
Isn't this true by default?

Reply:
No. That’s because IsVisible was false by default (IsVisible is the same as ShowItem)